stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/message/internal/MessageDao.kt

```diff
@@ -85,7 +85,7 @@ internal interface MessageDao {
     @Query(
         "SELECT * from $MESSAGE_ENTITY_TABLE_NAME " +
             "WHERE cid = :cid " +
-            "AND (createdAt > :dateFilter || createdLocallyAt > :dateFilter) " +
+            "AND (createdAt > :dateFilter OR createdLocallyAt > :dateFilter) " +
```

Comment:
Do we have the same issue of using `||` instead of `OR` in V5? =]

Reply:
No, this is just to keep the code using only words, not symbols. But both work.